webcomponents/vite.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from "vite";
+import { resolve } from "path";
+import dts from 'vite-plugin-dts'
+import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js'
+
+export default defineConfig({
+  build: {
+    minify: 'terser',
+    sourcemap: true,
+    lib: {
+      entry: resolve(__dirname, "index.ts"),  // 配置入口文件路径
```

Comment:
```suggestion
      entry: "index.ts",
```

`path.resolve()` seems unnecessary, just the relative path is enough. Also [`__dirname` is a CommonJS API](https://nodejs.org/docs/latest/api/modules.html#__dirname) which Vite would deprecate soon: https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated